### PR TITLE
Updates theme approach

### PR DESCRIPTION
**Pull request by mejackreed**
Inspired by embedding M3 in sul-embed, this PR builds off of discussion in #1906 and work completed in #1979 .

Based off [Material-UI documentation](https://material-ui.com/customization/themes/#type-light-dark-theme) my understanding is that setting `light` and `dark` primary values is used for setting the color when in a specific theme. This pull request resolves this by setting a new `lighten` and `darken` keys. While maybe not the final way we could do this, it is a refactor without modifying the colors currently being used.

Future alternatives could be to use the `lighten` and `darken` methods from Material-UI to do these calculations on the fly.

@ggeisler happy to chat more about this approach

### Before
Screen Shot 2019-03-29 at 10 06 08 AM

### After
Screen Shot 2019-03-29 at 10 06 18 AM

### Before
Screen Shot 2019-03-29 at 10 06 36 AM

### After
Screen Shot 2019-03-29 at 10 06 49 AM

### CODE REVIEW

**Comment by ggeisler** [comment]
@mejackreed Okay, great. I'll read up on the theme config stuff and make sure I understand it better, and check out the results from this PR in the themes. If @jvine and I think we need to still adjust colors, we can discuss.

File: src/config/settings.js [hunk 1]
```diff
@@ -8,13 +8,17 @@ export default {
...
         main: '#1967d2',
-        light: '#64b5f6',
-        dark: '#0d47a1',
+      },
+      darkened: { // custom colors used for a specific offset in some places
+        dark: '#424242',
+        light: '#eeeeee'
+      },
+      lightened: { // custom colors used for a specific offset in some places
+        dark: '#000000',
```
Comment (ggeisler) [inline src/config/settings.js]: It seems odd that (for the Dark theme, if I'm understanding things correctly) a "lightened" color would be `#000000` or black, while the "darkened" color would be `#424242`, which is an off-black. Should those be reversed?

I haven't yet examined the theme in detail to how or if these values are relevant so feel free to ignore this; it just seems on the surface that the "darkened" version should be darker than the "lightened" version.

Comment (mejackreed) [reply src/config/settings.js]: Yeah it totally could be changed.. this is really just a placeholder as my understanding was that dark theming hasn't really taken place yet.

Comment (ggeisler) [reply src/config/settings.js]: Sure, makes sense. If you don't mind making the update, I'd switch those two values just so we're starting with something that makes more logical sense (having the darkened variant darker than the lightened variant).

Comment (mejackreed) [reply src/config/settings.js]: Of course! I've made the update.

Comment (ggeisler) [reply src/config/settings.js]: Thanks Jack

**Pull request merged**